doc/internal/man3/bn_mul_words.pod

```diff
@@ -8,8 +8,9 @@ bn_sqr_comba4, bn_sqr_comba8, bn_cmp_words, bn_mul_normal,
 bn_mul_low_normal, bn_mul_recursive, bn_mul_part_recursive,
 bn_mul_low_recursive, bn_sqr_normal, bn_sqr_recursive,
 bn_expand, bn_wexpand, bn_expand2, bn_fix_top, bn_check_top,
-bn_print, bn_dump, bn_set_max, bn_set_high, bn_set_low - BIGNUM
-library internal functions
+bn_print,
```

Comment:
Turns out `bn_print` doesn't exist anymore either (removed in d459e39012e9912e5e827338741e7894e7b2a876). The one in x_bignum.c is unrelated and doesn't print to stdout.

Reply:
gone.  minor word changes ensued.